packages/babel-plugin/src/babel-plugin.tsx

```diff
@@ -48,19 +51,45 @@ export default declare<State>((api) => {
       this.cache = cache;
       this.includedFiles = [];
       this.pathsToCleanup = [];
+      this.pragma = {};
     },
     visitor: {
       Program: {
+        enter(_, state) {
+          const { file } = state;
+
+          if (file.ast.comments) {
+            for (const comment of file.ast.comments) {
+              const jsxSourceMatches = JSX_SOURCE_ANNOTATION_REGEX.exec(comment.value);
+              const jsxMatches = JSX_ANNOTATION_REGEX.exec(comment.value);
+
+              if (jsxSourceMatches && jsxSourceMatches[1] === COMPILED_MODULE) {
+                // jsxImportSource pragma found - turn on CSS prop!
```

Comment:
This may resolve https://github.com/atlassian-labs/compiled/issues/836

Reply:
I don't think it will, but haven't dug deeply into it